app/controllers/tasks_controller.rb

```
@@ -97,6 +97,10 @@ def update
     tasks_to_return = (QueueForRole.new(user_role).create(user: current_user).tasks + tasks).uniq
 
     render json: { tasks: json_tasks(tasks_to_return) }
+  rescue ActiveRecord::RecordInvalid => error
+    Raven.capture_exception(error)
```

Do we need to re-raise the error here / render something here?

---

I didnt want to lose the error, and the UI is not able to receive the error at the moment because it immediately redirects the user away from the page so there is not a way to show the error message right now without refactoring the modal and potentially parts of the QueueApp container

---

While the error can't be processed on the frontend, I think it would still be useful to see a status code from the network tab. I'm not actually sure what status code Rails returns when there's no render statement, but I'm guessing its a 200 (?).

---

The error I was receiving on the frontend in the network tab after adding the above line was a 422 (Entity not-processable)